src/lib.rs

```diff
@@ -39,11 +39,16 @@ pub struct VadSession {
     state: VadState,
     session_audio: Vec<f32>,
     processed_samples: usize,
+    deleted_samples: usize,
     silent_samples: usize,
+
     /// Current start of the speech in milliseconds
-    speech_start: Option<usize>,
+    speech_start_ms: Option<usize>,
     /// Current end of the speech in milliseconds
-    speech_end: Option<usize>,
+    speech_end_ms: Option<usize>,
+
+    /// Cached current active samples
+    cached_active_speech: Vec<f32>,
```

Comment:
I think this is a good compromise on keeping all the audio around and maintaining some of the old behaviour :+1: